Network/API/NetworkModel.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  NetworkModel.swift
+//  CleanArchitectureRxSwift
+//
+//  Created by Andrey Yastrebov on 10.03.17.
+//  Copyright © 2017 sergdort. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+import Domain
+import RxAlamofire
+import RxSwift
+import ObjectMapper
+
+private let ApiEndpoint = "https://jsonplaceholder.typicode.com"
+
+public final class NetworkModel {
+
+    func fetchPosts() -> Observable<[Post]> {
+        return RxAlamofire
+            .json(.get, ApiEndpoint + "/posts")
+            .debug()
+            .catchError { error in
```

Comment:
This is not a goo practice to catch error on domain layer. Also by returning `.never` you preventing sequence from completion, which could cause to weird behaviour in case this observable used some where else with operator which relying on completion of the sequence